bedrock/mozorg/templates/mozorg/projects/calendar.html

```diff
@@ -1,90 +0,0 @@
-{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The URL for this page and the one below can be removed:

https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/urls.py#L32-L34

It looks like we can also remove the view for holiday calendars:

https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/views.py#L155-L172

And the calendar data:

https://github.com/mozilla/bedrock/tree/master/media/caldata

Reply:
✅